test/sanity/validate-modules/validate-modules

```diff
@@ -851,6 +853,30 @@ class ModuleValidator(Validator):
 
         return False
 
+    def _check_for_subprocess(self):
+        for child in self.ast.body:
+            if isinstance(child, ast.Import):
```

Comment:
The reason for this is to avoid going through the file if there is no need. If you use `Popen` and not import it I guess it will fail much earlier than here. If you think that's not required I can remove it no probs. I can add the `ImportFrom` case otherwise